strabo/strabo/static/js/obsolete_interest_points_livy.js

```diff
@@ -0,0 +1,9 @@
+var lat_setting = 41.892695
```

Comment:
Moved livy interest points here & renamed to keep data here without allowing anything to read it.

Eventually, we'll delete this, but not yet.